packages/language-server/package.json

```diff
@@ -6,6 +6,7 @@
     "typings": "dist/src/index",
     "scripts": {
         "test": "cross-env TS_NODE_TRANSPILE_ONLY=true mocha --require ts-node/register \"test/**/*.ts\" --exclude \"test/**/*.d.ts\"",
+        "test2": "cross-env TS_NODE_TRANSPILE_ONLY=true mocha --require ts-node/register \"test/**/FindFileReferencesProvider.test.ts\" --exclude \"test/**/*.d.ts\"",
```

Comment:
Note - Use this for easy single testing. Remove when done

Reply:
Tip for next time: you can do `it.only(..)` or `describe.only` to achieve the same effect